insights/parsers/installed_rpms.py

```diff
@@ -17,7 +17,7 @@
 import warnings
 
 from ..util import rsplit
-from .. import parser, get_active_lines, CommandParser
+from .. import parser, CommandParser
```

Comment:
and since you updated the code here, could you please update the other `relative` importing to the full importing and sort them in alphabet order?

Reply:
updated